compiler/test/formatter_inputs/ifthenelse.gr

```diff
@@ -1,9 +1,22 @@
-if (true) { "a reallly long string long long long lfd dsds dsds dsds"} else if (false) "aaa" else { "short"}
+if (true) {
```

Comment:
@marcusroberts did you accidentally format this file?

Reply:
Urgh!  Yes I did.  I'll fix it